metadata.rb

```diff
@@ -22,7 +22,7 @@
 
 depends 'build-essential', '>= 6.0.0'
 depends 'chef-sugar', '>= 3.2.0'
-depends 'chef-ingredient', '>= 0.18.0'
+depends 'chef-ingredient', '>= 0.21.0'
```

Comment:
This will be updated to match the version given to `chef-ingredient` when a new version is released to supermarket.chef.io following the merge of https://github.com/chef-cookbooks/chef-ingredient/pull/128.